src/main/java/ca/solostudios/strata/parser/tokenizer/ParseException.java

```diff
@@ -35,7 +35,7 @@
 /**
  * Represents an error or a warning, which occurred when parsing an input.
  */
-public final class ParseException extends Exception {
+public final class ParseException extends RuntimeException {
     private static final long serialVersionUID = -2935358064424839548L;
```

Comment:
Given it's a serializable class, and it's changed, we should probably update the serialVersionUID